### PR TITLE
Added platinum grip and +1 additional effect of stun

### PR DESCRIPTION
**Pull request by dwn134**

### CODE REVIEW

File: sql/item_mods.sql [hunk 1]
```diff
@@ -23273,7 +23273,9 @@ INSERT INTO `item_mods` VALUES (19018,12,1);
 INSERT INTO `item_mods` VALUES (19018,13,1);
 INSERT INTO `item_mods` VALUES (19018,14,1);
 INSERT INTO `item_mods` VALUES (19019,25,2);
+INSERT INTO `item_mods` VALUES (19019,431,);
```
Comment (TeoTwawki) [inline sql/item_mods.sql]: missed a value

**Comment by dwn134** [comment]
Hm. Must have been a problem in my coping files to master. Becuase its on on my end

**Comment by teschnei** [comment]
if you're copying anything when making a pull request, you're not using git correctly

**Comment by dwn134** [comment]
I used upload file from my pc to my fork. I know im terible at git. I tried
doing the correct way and kept getting the did not exit cleanly message

**Comment by TeoTwawki** [comment]
use these, stop trying to upload via website:

https://git-for-windows.github.io/

https://tortoisegit.org/download/


Video:
https://www.youtube.com/watch?v=pp2S2lHjzZI
ignore his links, they are old because video is old. The SSH thing he shows you isn't required - tortioise can manage your password these days. All you need are the 2 download links I provided.

**Comment by TeoTwawki** [comment]
> stop trying to upload via website:

.

> Add files via upload

:man_facepalming:

**Pull request merged**